site/content/docs/reference/supported-clusters.md

```diff
@@ -37,3 +37,6 @@ token credential request API strategy by default.
 
 To choose the strategy to use with the concierge, use the `--concierge-mode` flag with `pinniped get kubeconfig`.
 Possible values are `ImpersonationProxy` and `TokenCredentialRequestAPI`.
+
+Do not use the command line option `--anonymous-auth=false` in the `kube-apiserver` CLI for a cluster that does not use `impersonation proxy`. This is because the `kube-apiserver` blocks unauthenticated access to `TokenCredentialRequest` API of the Concierge. 
+This does not matter while using `impersonation proxy`, which will allow these TokenCredentialRequests requests anyway.
```

Comment:
I know this page is inconsistent in where it uses monospaced font (markdown backticks), but let's try this, where it is only used to denote literal text of command names, CLI options, and resource configuration options.

```suggestion
Do not use the command line option `--anonymous-auth=false` in the `kube-apiserver` CLI for a cluster that does not use the impersonation proxy strategy. This is because the `kube-apiserver` blocks unauthenticated access to the TokenCredentialRequest API of the Concierge, which will prevent users from being able to authenticate. 
This does not matter while using the impersonation proxy strategy, which will allow these TokenCredentialRequests requests anyway.
```